### DISCUSSION TITLE
USDT whormole asset issue

### DISCUSSION THREAD

**Discussion by Tzar16** [wormhole-foundation/wormhole#2521] [General] [answered]
Hi there

I used the portal token bridge to bridge USDT from polygon to ethereum. I need to use the redeem solution because I had and issue.
Now I Have an USDT whormole asset and I cannot do anything with it.

Is there sometjing wrong ? what can i do to get a native ethereum USDT ?

THanks in advance

**Comment by aadam-10** [comment, marked as answer]
Hi @Tzar16 , this is something that the folks over at Discord can assist you with: https://discord.com/invite/wormholecrypto
